src/unix/linux_like/linux/mod.rs

```diff
@@ -5428,43 +5435,6 @@ cfg_if! {
                 spbufp: *mut *mut spwd,
             ) -> ::c_int;
 
-            pub fn mq_open(name: *const ::c_char, oflag: ::c_int, ...) -> ::mqd_t;
-            pub fn mq_close(mqd: ::mqd_t) -> ::c_int;
-            pub fn mq_unlink(name: *const ::c_char) -> ::c_int;
-            pub fn mq_receive(
-                mqd: ::mqd_t,
-                msg_ptr: *mut ::c_char,
-                msg_len: ::size_t,
-                msg_prio: *mut ::c_uint,
-            ) -> ::ssize_t;
-            pub fn mq_timedreceive(
-                mqd: ::mqd_t,
-                msg_ptr: *mut ::c_char,
-                msg_len: ::size_t,
-                msg_prio: *mut ::c_uint,
-                abs_timeout: *const ::timespec,
-            ) -> ::ssize_t;
-            pub fn mq_send(
-                mqd: ::mqd_t,
-                msg_ptr: *const ::c_char,
-                msg_len: ::size_t,
-                msg_prio: ::c_uint,
-            ) -> ::c_int;
-            pub fn mq_timedsend(
-                mqd: ::mqd_t,
-                msg_ptr: *const ::c_char,
-                msg_len: ::size_t,
-                msg_prio: ::c_uint,
-                abs_timeout: *const ::timespec,
-            ) -> ::c_int;
-            pub fn mq_getattr(mqd: ::mqd_t, attr: *mut ::mq_attr) -> ::c_int;
-            pub fn mq_setattr(
-                mqd: ::mqd_t,
-                newattr: *const ::mq_attr,
-                oldattr: *mut ::mq_attr
-            ) -> ::c_int;
-
-            pub fn pthread_mutex_consistent(mutex: *mut pthread_mutex_t) -> ::c_int;
```

Comment:
Did these functions move because they are now available on openharmony? If so, that should go in a different commit preferably.